### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ishristov**
MintGreen bitcoin mining project for heating North Vancouver

### CODE REVIEW

**Comment by DamienSomerset** [comment]
I'd like to suggest changing the scale metric of the document from xx miners to megawatts. Something like Small 1-10MW, Medium 10-100MW, Large 100+MW.

**Comment by DamienSomerset** [comment]
Biomass - Easy Crypto Hunter

TLDR: Apparently England went through a phase in which the government was offering incentives for farmers to install biogas digestors to generate power from farm waste, mainly manure. So many farmers ended up installing these devices that the need for energy dropped and the farmers were left with electricity the grid didn’t want. This company set them up with GPU miners that mine different proof of work blockchains, like Ravencoin, and the farmers can choose what crypto they would like to be paid in.

https://anchor.fm/compass-podcast/episodes/Farmers-are-Mining-Crypto-with-Cow-Poop--Josh-Riddett--Compass-Podcast-e12r9jf/a-a5t19f9

https://www.easycryptohunter.co.uk/choose-the-uks-mining-experts/

https://www.easycryptohunter.co.uk/testimonials/

https://www.farminguk.com/news/farmers-use-cow-manure-to-power-crypto-mining_58552.html

https://www.youtube.com/watch?v=JsdzesEv1q8

**Comment by DamienSomerset** [comment]
Heat Recapture & Hydro - Mint Green

Mint Green was founded by heat recapture specialists. They deconstruct  Bitcoin ASIC miners and put them in a device the circulates fluid to cool the boards. This fluid is run though a heat recapture device and used for other purposes, which include drying sea water in large vats in order to make sea salt, heating a whisky distillery, and I think they have a pilot that’s heating the streets of North Vancouver. Due to their location 90%+ of the power they use to run the miners comes from hydroelectric.

https://anchor.fm/compass-podcast/episodes/Does-Green-Bitcoin-Mining-Exist---Colin-Sullivan--Compass-Podcast-eutccv

https://mintgreen.co/

https://vancouversun.com/news/local-news/bitcoin-could-be-heating-homes-in-north-vancouver-next-year

https://www.independent.co.uk/news/world/americas/north-vancouver-bitcoin-mining-heating-b1941159.html

https://www.investing.com/news/cryptocurrency-news/north-vancouver-making-strides-to-become-worlds-first-city-heated-by-bitcoin-2647251

https://www.washingtonexaminer.com/news/canadian-city-becomes-first-world-heated-bitcoin-mining/

**Comment by arcbtc** [comment]
Wow, didn't see this pr. Thanks, will look through later
